ra_aid/agent_backends/ciayn_agent.py

```
@@ -17,8 +17,9 @@
 from ra_aid.prompts.ciayn_prompts import CIAYN_AGENT_SYSTEM_PROMPT, CIAYN_AGENT_HUMAN_PROMPT, EXTRACT_TOOL_CALL_PROMPT, NO_TOOL_CALL_PROMPT
 from ra_aid.tools.expert import get_model
 from ra_aid.tools.reflection import get_function_info
-from ra_aid.console.output import cpm
+from ra_aid.tool_configs import CUSTOM_TOOLS
 from ra_aid.console.formatting import print_warning, print_error, console
+from ra_aid.console.output import cpm
```

Ruff (0.8.2)

`ra_aid.console.output.cpm` imported but unused

Remove unused import: `ra_aid.console.output.cpm`

(F401)